kolibri/content/utils/annotation.py

```diff
@@ -195,6 +196,14 @@ def recurse_availability_up_tree(channel_id):
             )
         ).where(ContentNodeTable.c.id == child.c.parent_id)
 
+        coach_content_nodes = select([all_(child.c.coach_content)]).where(
+            and_(
+                child.c.available == True,  # noqa
+                child.c.level == level,
+                child.c.channel_id == channel_id,
+            )
+        ).where(ContentNodeTable.c.id == child.c.parent_id).as_scalar()
+
```

Comment:
I'm still learning to process SQLAlchemy efficiently in my head -- could you add one-liner comments above these to explain their purpose, to support the reader? (e.g. took me a min to realize this wasn't being executed directly, and instead only embedded down below)